plutus-playground-server/src/Playground/Usecases.hs

```diff
@@ -1,18 +1,28 @@
-{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell   #-}
 
 module Playground.Usecases where
 
-import           Data.ByteString (ByteString)
-import           Data.FileEmbed  (embedFile, makeRelativeToProject)
+import           Data.ByteString    (ByteString)
+import qualified Data.ByteString    as BS
+import           Data.FileEmbed     (embedFile, makeRelativeToProject)
+import qualified Data.Text          as T
+import qualified Data.Text.Encoding as T
 
-vesting :: ByteString
-vesting = $(makeRelativeToProject "usecases/Vesting.hs" >>= embedFile)
+marker :: T.Text
+marker = "TRIM TO HERE"
```

Comment:
it's a bit ugly, couldn't we just trim to the end of the `module .* where`?

Reply:
I didn't want to hack up some regex thing, and I'm not completely confident that going to the end of the first "where" won't get the wrong thing. I agree this is kind of ugly but it's also very simple :laughing:

Reply:
yes I suppose the alternative is probably regex and therefore dangerous. I leave it to you because I'm on the fence.

Reply:
I propose we leave this for now and if it is annoying/dangerous we fix it.